bookwyrm/templates/components/modal.html

```diff
@@ -1,8 +1,13 @@
-<div class="modal hidden" id="{{ controls_text }}-{{ controls_uid }}">
+<div
```

Comment:
accesslint keeps flagging divs like this, it seems like a bug with the bot to me? I'm going to look into alternate accessibility linters

Reply:
Do you know why the error is triggered? Does the linter parse the resulting HTML or the template? If that’s the template, maybe the variables `{{ … }}` are triggering the error.

Reply:
Adding sections is not necessarily anodyne as screen readers will present them as a menu to browse with. Adding a label should allow to present more informative structure to screen readers.

This is an assumption, but if there is plenty of sections without labels, users of screen readers will probably suffer more than benefit from them.

Reply:
In that case, the ID were still duplicated because there are several modals for the same status and I did not include the action in the ID, so I guess `aria-labelledby` didn’t know which one to use.

It looks happy now, right? :)

Reply:
gotcha, thanks for explaining :)